### DISCUSSION TITLE
Server.xml is missing

### DISCUSSION THREAD

**Discussion by qwerty13** [OvenMediaLabs/OvenMediaEngine#1110] [Help]
Hi,
I'm using OME v0.14.14 on Ubuntu 22.04 Server and OME is running on Docker. I previously Configured the "Server.xml" file and note it for myself that it is in "var/lib/docker/volumes/ome-origin-conf/_data/Server.xml", but now not this address nor other addresses mentioned in OME Documentation does not exist! How could this happen?

(
I restart OME, it works but  there is still no xml file. I Search for "Server.xml" in entire system and found these:
/# find / -iname "Server.xml"

/var/lib/docker/overlay2/ec86bd2e072a7b94e7739c58e15cca252489cb4250f59871c51b7abd09036a81/diff/opt/ovenmediaengine/bin/edge_conf/Server.xml

/var/lib/docker/overlay2/ec86bd2e072a7b94e7739c58e15cca252489cb4250f59871c51b7abd09036a81/diff/opt/ovenmediaengine/bin/origin_conf/Server.xml

/var/lib/docker/overlay2/8657b9f234990bf312c5f28e1b1ce60c0df61157a6accfcf45618abfb5934bba/diff/opt/ovenmediaengine/bin/edge_conf/Server.xml

/var/lib/docker/overlay2/8657b9f234990bf312c5f28e1b1ce60c0df61157a6accfcf45618abfb5934bba/diff/opt/ovenmediaengine/bin/origin_conf/Server.xml

/var/lib/docker/overlay2/0b3e86d8a3ab24f15b2f6e21b692f7bbe8abc8a61d9da44462c4272068db4785/merged/opt/ovenmediaengine/bin/origin_conf/Server.xml

/var/lib/docker/overlay2/0b3e86d8a3ab24f15b2f6e21b692f7bbe8abc8a61d9da44462c4272068db4785/merged/opt/ovenmediaengine/bin/edge_conf/Server.xml
)

**Comment by bchah** [comment]
OME in Docker will use the file at '/opt/ovenmediaengine/bin/origin_conf/Server.xml' if there is no additional configuration provided. Try editing or replacing the file at that location, or map the folder to it in your Docker run command like:

-v /path/to/myfoldercontainingserverxml:/opt/ovenmediaengine/bin/origin_conf

You can also restart OME itself, give it a -c flag and pass in the Server.xml location if it is different from default. But the Docker version does this already to the origin_conf folder by default.
